docs/source/release-notes.html.md

```diff
@@ -7,6 +7,12 @@ weight: 8
 
 If you have any questions or comments about these notes, please contact DfE via Slack or email.
 
+## 16 February 2024
+
+We’ve found and fixed a bug that meant for a small number of NPQ applications the value in the ‘itt_provider’ field was incorrectly set, so was not showing the name of the provider.
```

Comment:
Hi @pmanrubia Believe ‘itt_provider’ should be with ![Screenshot 2024-02-16 at 13 21 45](https://github.com/DFE-Digital/early-careers-framework/assets/3020581/700cf2ed-2d94-49d6-83f0-d1a2a3791b5b) instead in order to be rendered as ![Screenshot 2024-02-16 at 13 19 18](https://github.com/DFE-Digital/early-careers-framework/assets/3020581/2cdfeec8-e320-4e1c-954f-06c7bb9575de)